frontend/src/assets/content/arp/contact-information/contact-us.md

```diff
@@ -21,12 +21,12 @@ Granby, CO 80446
 
 email: [srdvis@fs.fed.us](mailto: srdvis@fs.fed.us)
 
-Permits are only available for purchase in person on weekdays during the Christmas tree cutting season.
+Permits are available for purchase in person on weekdays during the Christmas tree cutting season.
```

Comment:
would we want to add "for this district"?

Reply:
Good question, I don't think we need to since the information is below the corresponding district.

Reply:
Also looks like we might have tackled some of these changes here: https://github.com/18F/fs-open-forest-platform/pull/402/files

Can you confirm?